css-scroll-snap-2/Overview.bs

```diff
@@ -279,13 +279,35 @@ Snap Events {#snap-events}
 				<th>Event handler event type
 		<tbody>
 			<tr>
-				<th><dfn event>snapChanged</dfn>
+				<th><dfn event>snapchanged</dfn>
 				<td>{{scroll!!event}}
 			<tr>
-				<th><dfn event>snapChanging</dfn>
+				<th><dfn event>snapchanging</dfn>
 				<td>{{scroll!!event}}
 	</table>
 
+	{{ snapchanged }} will indicate that the set of snap areas to which a snap container is snapped has changed.
+	{{ snapchanging }} will hint at the set of snap areas to which a snap container would snap during a scroll.
```

Comment:
When are these events dispatched?

Reply:
Thanks for pointing out this gap. snapchanged should fire at the end of a scroll (or style/layout change that triggers snapchanged) and before a scrollend event. snapchanging should fire during a scroll, before scroll events. I've included these details in the spec.

Reply:
For `scroll` and `scrollend`, it's defined here:

https://drafts.csswg.org/cssom-view-1/#scrolling-events

which integrates closely with HTML's event loop:

https://html.spec.whatwg.org/multipage/webappapis.html#update-the-rendering

If new events are to be fired at `scroll`-event timing, it should be in cssom-view's algo so that it's well-defined when they fire. But also see https://github.com/w3c/csswg-drafts/issues/7442#issuecomment-1824851458